CONTRIBUTING.md

```diff
@@ -8,98 +8,96 @@ are important to the project's success.
 
 ## The contribution process at a glance
 
-1. Read the [README.md file](README.md).
-2. If you want, set up your environment to be able to [run tests](README.md#running-the-tests).
+1. Find out [where to make your changes](#where-to-make-changes).
```

Comment:
Maybe we can more prominently mention a) where / how to ask for help, b) how to get started contributing (good-first-issue label / link to #4641 )

Reply:
There is a section about where to get help in the README. That said, I'm sure this can be improved, but I'd suggest to move this to a separate PR.